### ISSUE TITLE
Crash after running step

### ISSUE THREAD

**Issue by thboop** [fhammerl/runner-hooks#69]
Got a crash when running a step hook after returning. Might be related to #65 

```
[2022-05-04 19:01:56Z ERR  StepsRunner] Caught exception from step: System.NullReferenceException: Object reference not set to an instance of an object.
   at GitHub.Runner.Worker.Container.ContainerHooks.ContainerHookManager.ScriptStepAsync(IExecutionContext context, ContainerInfo container, String entryPointArgs, String entryPoint, IDictionary`2 environmentVariables, String prependPath, String workingDirectory)
   at GitHub.Runner.Worker.Handlers.ContainerStepHost.ExecuteAsync(IExecutionContext context, String workingDirectory, String fileName, String arguments, IDictionary`2 environment, Boolean requireExitCodeZero, Encoding outputEncoding, Boolean killProcessOnCancel, Boolean inheritConsoleHandler, String standardInInput, CancellationToken cancellationToken)
   at GitHub.Runner.Worker.Handlers.ScriptHandler.RunAsync(ActionRunStage stage)
   at GitHub.Runner.Worker.ActionRunner.RunAsync()
   at GitHub.Runner.Worker.StepsRunner.RunStepAsync(IStep step, CancellationToken jobCancellationToken)
[2022-05-04 19:01:56Z INFO StepsRunner] Step result: Failed
```

**Comment by ruvceskistefan** [comment]
Can you check if it happens after committing changes that closed issue #65?

**Comment by thboop** [comment]
That fixed it, thanks @ruvceskistefan !
